src/components/Select.vue

```diff
@@ -213,6 +214,16 @@
   .v-select.unsearchable input[type="search"]:hover {
     cursor: pointer;
   }
+  .v-select input[type="search"].hidden {
+    display: none;
```

Comment:
In my testing, `display: none` can mess up the vertical alignment of the clear button and the dropdown arrow. Perhaps we consider the following instead, as they preserve the height of the input element, and the vertical alignment of the other elements?

```css
width: 0px;
padding: 0;
```

Reply:
Good shout @stevenharman , will update